### PR TITLE
[BD] Fix typos and improve wording

### PR DESCRIPTION
**Pull request by jjasferreira**

### CODE REVIEW

File: content/bd/0001-intro-dbms.md [hunk 1]
```diff
@@ -23,44 +23,44 @@ entre [**bases de dados**](color:green) e [**sistemas de gestão de bases de dad
...
 Por exemplo, numa instituição como o Técnico, é preciso associar professores a um
 departamento. Teremos então duas [**entidades**](color:pink): professores e departamentos.
 Estas duas entidades estão [**relacionadas**](color:purple): o professor _X_ pertence ao departamento _Y_.
 
 Por outro lado, um [**sistema de gestão de bases de dados**](color:yellow) é um pacote de
 software que está desenhado para guardar e gerir bases de dados.
 Existem diferentes tipos de SGBDs, como veremos mais à frente.  
-Os SGBD têm de uma grande quantidade de funcionalidades para facilitar a vida a
-quem usa as bases de dados (e.g. programadores), como garantias de consistência,
+Os SGBDs têm de uma grande quantidade de funcionalidades para facilitar a vida a
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:34]:
```suggestion
Os SGBDs têm uma grande quantidade de funcionalidades para facilitar a vida a
```

File: content/bd/0001-intro-dbms.md [hunk 2]
```diff
@@ -23,44 +23,44 @@ entre [**bases de dados**](color:green) e [**sistemas de gestão de bases de dad
...
+quem usa bases de dados (e.g. programadores), como garantias de consistência,
 controlo de acessos, gestão de concorrência (_transactions_), e muito mais.  
 Estas são também vantagens da utilização de SGBDs em vez de [**ficheiros**](color:red),
-visto que nesse caso um programador necessitaria de programar estas funcionalidades do
-zero todas as vezes, enquanto que SGBDs conseguem abstrair toda essa complexidade.
+visto que, nesse caso, um programador necessitaria de programar estas funcionalidades a partir do
+zero todas as vezes, enquanto que os SGBDs conseguem abstrair toda essa complexidade.
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:39]:
```suggestion
zero todas as vezes, enquanto que SGBDs conseguem abstrair toda essa complexidade.
```
Generic reference

File: content/bd/0001-intro-dbms.md [hunk 3]
```diff
@@ -23,44 +23,44 @@ entre [**bases de dados**](color:green) e [**sistemas de gestão de bases de dad
...
+Hoje em dia, existem também serviços cloud que nos dão acesso a bases de dados
 de uma forma extremamente simples e escalável, como a Amazon Web Services e
 a Azure Cloud.
 
 ## Vantagens e Desvantagens de SGBDs
 
 [**Vantagens**](color:green)
 
-Já referenciámos algumas das vantagens de SGBDs, no entanto existem muitas mais.
+Já referenciámos algumas das vantagens dos SGBDs, no entanto existem muitas mais.
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:58]:
```suggestion
Já referenciámos algumas das vantagens de SGBDs, no entanto existem muitas mais.
```

File: content/bd/0001-intro-dbms.md [hunk 4]
```diff
@@ -23,44 +23,44 @@ entre [**bases de dados**](color:green) e [**sistemas de gestão de bases de dad
...
+cada um com as suas vantagens e desvantagens, entre os quais:
 MySQL, MariaBD, PostgreSQL, SQLite, Microsoft SQL Server, Oracle Database, H2,
-Microsoft Access, e muitos outros.  
+Microsoft Access, e muitos outros.
 Todos estes são exemplos de SGBDs relacionais. Existem outros tipos de SGBDs,
-entre os quais NoSQL, os quais iremos explorar no futuro.
+como é o caso do _NoSQL_, sobre os quais iremos falar no futuro.
 
-Hoje em dia existem também serviços cloud que nos dão acesso a bases de dados
+Hoje em dia, existem também serviços cloud que nos dão acesso a bases de dados
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:50]:
```suggestion
Para além disso, atualmente existem também serviços cloud que nos dão acesso a bases de dados
```
Avoid repetition (two `Hoje em dia`s in a row)

File: content/bd/0001-intro-dbms.md [hunk 5]
```diff
@@ -228,33 +228,33 @@ representa um único aluno.
...
-| ist1123456 | Diogo          | 2002-03-06 |
-| ist1123234 | Rafa           | 2002-08-05 |
-| ist1124453 | Tomás          | 2002-12-18 |
-| ist1123534 | Diogo          | 2001-09-16 |
-| ist1123532 | Diogo          | 2002-04-27 |
+| ist123456 | Diogo          | 2002-03-06 |
+| ist123234 | Rafa           | 2002-08-05 |
+| ist124453 | Tomás          | 2002-12-18 |
+| ist123534 | Diogo          | 2001-09-16 |
+| ist123532 | Diogo          | 2002-04-27 |
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:235]:
```suggestion
| ist1123456 | Diogo          | 2002-03-06 |
| ist1123234 | Rafa           | 2002-08-05 |
| ist1124453 | Tomás          | 2002-12-18 |
| ist1123534 | Diogo          | 2001-09-16 |
| ist1123532 | Diogo          | 2002-04-27 |
```
It does seem easier to read at first glance without the leading `1`, but student IST IDs always start with a `1` and the `123...` pattern is instinctively understood as a placeholder, which would make this informally-expressed regular expression technically incorrect.

This is definitely not that important though.

File: content/bd/0001-intro-dbms.md [hunk 6]
```diff
@@ -228,33 +228,33 @@ representa um único aluno.
...
 utilizado o [**SQL**](color:orange) (Structured Query Language).
 
-Na imagem abaixo podemos ver o esquema que representa a estrutura todo o fluxo de execução
+Na imagem abaixo, podemos ver o esquema que representa a estrutura de todo o fluxo de execução
 entre a aplicação e a base de dados, passando pelo SGBD.
 
 ![Esquema que representa a arquitetura de um SGBD](./assets/0001-dbms-architecture.svg#dark=3)
 
-Quando executamos uma _query_ (interrogação), esta é enviada ao
+Quando executamos uma _query_ (interrogação), esta é enviada à
```
Comment (RafDevX) [inline content/bd/0001-intro-dbms.md:256]:
```suggestion
Quando executamos uma _query_ (interrogação), esta é enviada ao
```
`Engine` (= `motor`) is arguably a masculine word in Portuguese. Think `pesquisei no [motor de busca] Google]` (vs `a [empresa] Google despediu 100 pessoas`), `a configuração do nginx`, etc.)

**Review by RafDevX — CHANGES_REQUESTED** [review]

**Review by diogotcorreia — APPROVED** [review]

**Review by RafDevX — APPROVED** [review]

**Pull request merged**